test/json_unittest.cc

```diff
@@ -307,24 +411,26 @@ TEST(EdgeTest, NegativeNumbers) {
 
 // Big tests.
 
+const char kComplexExample[] =
```

Comment:
`constexpr const char kComplexExample[]`. Also below.

Reply:
Done.

Reply:
You need both, unfortunately. `constexpr` refers to the array, and `const` refers to the `char`.